ci/run_envoy_docker.sh

```diff
@@ -18,7 +18,8 @@ USER_GROUP=root
 
 mkdir -p "${ENVOY_DOCKER_BUILD_DIR}"
 # Since we specify an explicit hash, docker-run will pull from the remote repo if missing.
-docker run --rm -t -i -u "${USER}":"${USER_GROUP}" -v "${ENVOY_DOCKER_BUILD_DIR}":/build \
+docker run --rm -t -i -e http_proxy=${http_proxy} -e https_proxy=${https_proxy} \
```

Comment:
Did you determine that all caps is no longer needed by Bazel?

Reply:
Ah, I see, you say it works. Fine, SGTM.

Reply:
generally http_proxy and https_proxy would already be exported for such build environment. I added an example as you suggested.